index.html

```diff
@@ -0,0 +1,142 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>Apple</title>
+    <link rel="stylesheet" href="style.css">
```

Comment:
**[REQUIERED]** As the [ validator ](https://validator.w3.org) is telling, The[ character encoding](https://www.w3schools.com/html/html_charset.asp) was not declared.

Reply:
@samgaco ,
You are right and thanks again ^^ Remember all details, sometimes, is really hard ^^ Trying to get used to it with time. But as far as you show me my mistakes or suggesting better ways, i will try to follow. I will try to fix all you mentioned above before submitting project of course.

Reply:
@samgaco,
Also i should say, 

> 
> 
> **[REQUIERED]** As the [ validator ](https://validator.w3.org) is telling, The[ character encoding](https://www.w3schools.com/html/html_charset.asp) was not declared.

I validate my html code in same website, but i do it by "Direct Input". I select all code(ctrl+A) then paste it as input and check validation. I followed same path to validate and if you try that same way, you can see that error does not appear at all. I am not sure about why but reason of not correcting that error is, did not see at all ^^

Reply:
Now it is all fixed except repetitive commits. Sorry about that really, i was working on some little details of/or same section/part/element/attribute or etc. and those moments are when i use same commits. I will try to write more clear and specific commits any more ^^